# 아래 코드 실행 시 오류가 발생한 사유를 설명해주세요

# t = (1, 2, 3)
# t[0] = 'a'
# Traceback (most recent call last):
# File "<pyshell#46>", line 1, in
# t[0] = 'a'
# TypeError: 'tuple' object does not support item assignment




    # 튜플은 변경 불가능한 자료형이기 때문에 내용 변경 불가